docs/using/installing/index.rst

```diff
@@ -557,6 +557,22 @@ The configuration item to use AWS should look like:
 Make sure that the ``region`` and ``zone`` match each other and that both match the region and zone where the Flocker agent nodes run.
 AWS must be able to attach volumes created in that availability zone to your Flocker nodes.
 
+CentOS 7
+........
+
+Run the following commands to enable the agent service:
+
+.. task:: enable_flocker_agent centos-7 ${CONTROL_NODE}
+   :prompt: [root@agent-node]#
+
+Ubuntu
+......
+
+Run the following commands to enable the agent service:
+
+.. task:: enable_flocker_agent ubuntu-14.04 ${CONTROL_NODE}
+   :prompt: [root@agent-node]#
+
```

Comment:
This is an improvement but this placement still doesn't quite make sense to me.  These things are still at the same level as the storage driver configuration documentation.  It seems like they belong in a different section or that the individual storage driver sections should be demoted to an inferior heading level.  I was also surprised to **return** to Flocker service configuration instructions after leaving the earlier Flocker installation section and spending time reading about `flocker-ca`.  I see why things happen in this order but I don't think that makes it less confusing to the user. :/

This whole document (and probably the activities it **documents**) probably needs to be revisited.  But for now maybe a change that makes sense is to:

  * Create a "Configuring the Flocker agent" section and put all of the agent.yml documentation beneath it
  * Create an "Enabling the Flocker agent service" section and put the `systemctl ...` etc stuff beneath that.